debezium-connector-oracle/src/main/java/io/debezium/connector/oracle/logminer/LogMinerMetrics.java

```diff
@@ -90,7 +103,7 @@ public void reset() {
         totalBatchProcessingDuration.set(Duration.ZERO);
         maxBatchProcessingThroughput.set(0);
         lastBatchProcessingDuration.set(Duration.ZERO);
-        networkConnectionProblemsCounter.set(0);
+        networkConnectionProblemsCounter.set(0);       
```

Comment:
```suggestion
        networkConnectionProblemsCounter.set(0);
```
Lets avoid the trailing spaces.